templates/frameworks/list.html.twig

```diff
@@ -117,8 +117,8 @@
                 {% if (guided_match_flag is defined) and (query is defined and query in keywords) and (pagination.totalResults > 1) %}
                     <div class="govuk-grid-column-two-thirds guided-match-background govuk-!-margin-bottom-2 guided-match-area">
                             <h2 class="govuk-heading-l ccs-font-weight-semibold text-white">Get help finding the right framework</h2>
-                            <h2 class="govuk-heading-s ccs-font-weight-semibold text-white">There are {{ pagination.totalResults }} results found for "{{ query }}". Depending on your requirements, some of these frameworks maybe better suited than others.</h2>
-
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold text-white">We have found {{ pagination.totalResults }} agreements based on your search "{{ query }}".</h2>
+                            <h3 class="govuk-heading-s ccs-font-weight-semibold text-white">If your search results contain multiple agreements, answer come simple questions first to help find the best one for your needs</h3>
```

Comment:
2 h2 classes will be flagged as incorrect in accessibility testing. These are also not headings, we should use body classes.